src/lib.rs

```diff
@@ -1538,7 +1544,7 @@ mod test {
 
     mod arith {
         use super::super::{Ratio, Rational};
-        use super::{_0, _1, _1_2, _2, _3_2, _NEG1_2, to_big};
+        use super::{to_big, _0, _1, _1_2, _2, _3_2, _NEG1_2};
```

Comment:
My VS-code's auto-formatter did this, but can I just leave it?

Reply:
As long as the CI `rustfmt` job accepts that, it's fine.